lbz/events/broker.py

```diff
@@ -2,23 +2,26 @@
 from typing import Callable, Dict, List
 
 from lbz.events.event import Event
+from lbz.handlers import BaseHandler
 from lbz.misc import get_logger
 
 logger = get_logger(__name__)
 
 
 # TODO: type_key and data_key will be const for EventBridge and different set for Cognito Events
-class EventBroker:
+class EventBroker(BaseHandler):
     def __init__(
         self,
-        mapper: Dict[str, List[Callable]],
-        raw_event: dict,
+        mapper: Dict[str, List[Callable[[Event], None]]],
+        event: dict,
+        context: object = None,
```

Comment:
According to [#989723417](https://github.com/pdyba/lambdalizator/pull/71#discussion_r989723417), `context` cannot be optional.